app/inputs/transaction_chosen_input.rb

```diff
@@ -21,7 +26,7 @@ def label
   private
 
   def collection_items
-    template.instance_variable_get("@#{attribute_name}")
+    template.instance_variable_get("@#{attribute_name}").to_a
```

Comment:
Does the `#to_a` need to be here given line 11? Or does it maybe not need to be on line 11, and be here instead.